cmd/migrate/migrate.go

```diff
@@ -89,6 +89,28 @@ func main() {
 		}
 	}
 
+	//Deletes Duplicate users from users table
+	if tx.Migrator().HasTable(&models.UserIdentity{}) {
+
+		var duplicates []models.UserIdentity
+		tx.Model(&models.UserIdentity{}).Select("AccountId").Group("AccountId").Having("COUNT(*) > 1").Find(&duplicates)
+
+		for _, dup := range duplicates {
+
+			var usersToDelete []models.UserIdentity
+			tx.Where("AccountId = ?", dup.AccountId).Find(&usersToDelete)
```

Comment:
I think we also want to sort them and keep the "last updated" and remove the rest.